tests/system/presenter/test_get_user_related_models.py

```diff
@@ -173,24 +183,72 @@ def test_get_user_related_models_no_permissions(self) -> None:
         self.set_models(
             {
                 "user/1": {"organization_management_level": None, "meeting_ids": [1]},
-                "meeting/1": {"name": "test"},
+                "committee/1": {"meeting_ids": [1]},
+                "meeting/1": {
+                    "name": "test",
+                    "is_active_in_organization_id": 1,
+                    "committee_id": 1,
+                },
                 "motion_submitter/2": {"user_id": 1, "meeting_id": 1},
             }
         )
         status_code, _ = self.request("get_user_related_models", {"user_ids": [1]})
         self.assertEqual(status_code, 403)
 
-    def test_get_user_related_models_permission_because_no_meeting_included(
+    def test_get_user_related_models_empty_meeting(
         self,
     ) -> None:
         self.set_models(
             {
                 "user/2": {"organization_management_level": None, "meeting_ids": [1]},
-                "meeting/1": {"name": "test"},
+                "committee/1": {"meeting_ids": [1]},
+                "meeting/1": {
+                    "name": "test",
+                    "is_active_in_organization_id": 1,
+                    "committee_id": 1,
+                },
             }
         )
-        status_code, _ = self.request("get_user_related_models", {"user_ids": [2]})
+        status_code, data = self.request("get_user_related_models", {"user_ids": [2]})
         self.assertEqual(status_code, 200)
+        assert data == {
+            "2": {
+                "meetings": [
+                    {
+                        "id": 1,
+                        "name": "test",
+                        "is_active_in_organization_id": 1,
+                    }
+                ]
+            },
+        }
+
+    def test_get_user_related_models_archived_meeting(
+        self,
+    ) -> None:
+        self.set_models(
+            {
+                "user/2": {"organization_management_level": None, "meeting_ids": [1]},
+                "committee/1": {"meeting_ids": [1]},
+                "meeting/1": {
+                    "name": "test",
+                    "committee_id": 1,
+                },
+            }
+        )
+        status_code, data = self.request("get_user_related_models", {"user_ids": [2]})
+        self.assertEqual(status_code, 200)
+        assert data == {
+            "2": {
+                "meetings": [
+                    {
+                        "id": 1,
+                        "name": "test",
+                        "is_active_in_organization_id": None,
+                    }
+                ]
+            },
+        }
```

Comment:
The result still misses the `organization_management_level: OML-String,` info, which is mentioned in the presenter doc. The existing test **test_get_user_related_models_permissions** , beginning at line 279, should have such a result entry, but it is missed and not checked.

Reply:
Somehow I thought that the presenter already returned the OML. Added it now